### PR TITLE
Log error instead of raising when using get_file

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11984

## Description
When using get_file on a malformed yml file, the error raised contained the tag <file>. logging this error when handling the exception raised another exception since the logger command thought this tag is something the logger should parse. In this pr, I replace the <file> tag with the actual file path before sending it to the logger, making it more readable and less troublesome. I assert no error is raised anymore.

### CODE REVIEW

**Comment by DinaMeylakh** [comment]
TDD: failing ut in [ecfa300](https://github.com/demisto/demisto-sdk/pull/4625/commits/ecfa30035d6e0a3adaa4779afffd0c99a3aa22f4) can be found in this [job](https://github.com/demisto/demisto-sdk/actions/runs/11442021118/job/31831445582?pr=4625)

**Comment by dorschw** [comment]
see https://github.com/demisto/demisto-sdk/pull/4631

**Comment by DinaMeylakh** [comment]
Filing ut can be found [here](https://github.com/demisto/demisto-sdk/actions/runs/11592683993/job/32275131278?pr=4625) in this commit [d94de5a](https://github.com/demisto/demisto-sdk/pull/4625/commits/d94de5a2072618cb6bfaf88f70feb6e28a3ef4d9)

File: demisto_sdk/commands/common/tests/tools_test.py [hunk 1]
```diff
@@ -190,6 +190,39 @@ class TestGenericFunctions:
...
+        # Since Github actions do not allow colors, the exception is usually raised locally only.
+        # To verify, you can delete this line and run the test locally and compare with the Github action.
+        mocker.patch(
+            "demisto_sdk.commands.common.tools.logger.error",
+            side_effect=raise_loguru_exp,
+        )
+        try:
+            get_file(file_path="some_file.yml", raise_on_error=False, clear_cache=True)
+        except Exception as e:
+            assert False, f"Function get_file raised an error: {e}"
```
Comment (dorschw) [inline demisto_sdk/commands/common/tests/tools_test.py:224]: use `with pytest.raises(Exception): ...` instead of `try/except`ing

**Review by dorschw — APPROVED** [review]

**Pull request merged**